extension.json

```diff
@@ -29,6 +29,18 @@
                 "type": "string",
                 "minLength": 1
               },
+              "stagingEdgeConfigId": {
+                "type": "string",
+                "minLength": 1
+              },
```

Comment:
Is there a different platform api host for staging and development?

Reply:
There may be, but their staging and development is for them to test new changes.  Blackbird holds different environment configurations to match nicely with the launch build environments.  https://git.corp.adobe.com/pages/experience-edge/blackbird/#/api/environments